.travis.yml

```diff
@@ -1,6 +1,9 @@
 language: python
 python:
   - "3.6"
+before_install:
+  - sudo apt-get -qq update
+  - sudo apt-get install -y portaudio19-dev libffi-dev libssl-dev
```

Comment:
I think you mentioned recently that you thought we didn't need libffi-dev anymore since we dropped the pyaudio dep... happy to still install it, but even happier to remove it :)